test/spec/modules/improvedigitalBidAdapter_spec.js

```diff
@@ -150,6 +150,14 @@ describe('Improve Digital Adapter Tests', function () {
   });
 
   describe('interpretResponse', () => {
+    let registerSyncSpy;
+    beforeEach(() => {
+      registerSyncSpy = sinon.stub(userSync, 'registerSync');
```

Comment:
since you're stubbing, this should be called `registerSyncStub` to prevent confusion.  spies and stubs behave differently in sinon.

Reply:
Done